app/controllers/admin/dfc_product_imports_controller.rb

```diff
@@ -40,6 +40,9 @@ def index
            ActionController::ParameterMissing => e
       flash[:error] = e.message
       redirect_to admin_product_import_path
+    rescue Rack::OAuth2::Client::Error
+      flash[:error] = t(".connection_invalid")
```

Comment:
I am wondering if it's worth logging the error ?

Reply:
I would wait until we actually need the data. Otherwise it's just noise in Bugsnag.